frontend/src/metabase/setup/components/DatabaseHelp/DatabaseHelp.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { DatabaseHelpRoot, DatabaseHelpCard } from "./DatabaseHelp.styled";
+
+interface Props {
+  engine?: string;
+  isStepActive: boolean;
+}
+
+const DatabaseHelp = ({ engine, isStepActive }: Props): JSX.Element => {
+  const isVisible = isStepActive && engine != null;
+
+  return (
+    <DatabaseHelpRoot
+      isVisible={isVisible}
+      data-testid="database-setup-help-card"
+    >
+      <DatabaseHelpCard {...{ engine, hasCircle: false }} />
```

Comment:
Workaround for some TS issues. Otherwise I'd need to migrate `AddDatabaseHelpCard`